README.md

```diff
@@ -78,8 +78,39 @@ To run the live DiceDB server for local development:
 ```sh
 $ git clone https://github.com/dicedb/dice
 $ cd dice
-$ air
-```
+
+$ # set up configuration file # (optional but recommended)
+$ sudo mkdir -p /etc/dice
+$ sudo chown root:$USER /etc/dice
+$ sudo chmod 775 /etc/dice # or 777 if you are the only user
+$ go run main.go -init-config
```

Comment:
Let's avoid having the configuration-related commands here in the main setup instructions (people should be exposed to the minimum amount of info required to launch the server with as few commands as possible).

You can add a sub heading under this heading called `Local Setup with Custom Config` and add these instructions there.